city_scrapers/spiders/chi_ssa_51.py

```diff
@@ -0,0 +1,75 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa51Spider(CityScrapersSpider):
+    name = "chi_ssa_51"
+    agency = "Chicago Special Service Area #51 Chatham"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.cbatechworks.org"]
+    start_urls = ["http://www.cbatechworks.org/"]
+    location = {"address": "806 East 78th Street, Chicago IL 60619", "name": "QBG Foundation"}
+    last_parsed_date = None
```

Comment:
It looks like this isn't referenced anywhere outside of the `parse` and `_parse_date` function, and we mainly use this to maintain state between requests. It might be a little cleaner to declare this variable inside the `parse` function, and have `_parse_date` return both the parsed date and `last_parsed_date`, updating it if it has changed.